components/resources/ai_chat_ui_strings.grdp

```diff
@@ -303,6 +303,18 @@
   <message name="IDS_CHAT_UI_SHOW_DETAILS_BUTTON_LABEL" desc="A button label to show details of the AI Assistant reasoning">
     Show details
   </message>
+  <message name="IDS_CHAT_UI_OPEN_LABEL" desc="A button label to open a link">
+    Open
+  </message>
+  <message name="IDS_CHAT_UI_OPEN_EXTERNAL_LINK" desc="A button label to open a link">
+    Open external link?
+  </message>
+  <message name="IDS_CHAT_UI_OPEN_EXTERNAL_LINK_INFO" desc="Info text to open a link">
+    Leo AI wants to open a website outside of the application. Do you want to continue?
```

Comment:
Would `application` be confusing here? Makes me think it will open outside of Brave. Maybe outside of the chat?

Reply:
Updated this to `Leo AI wants to open a website in a new tab. Do you want to continue?`